src/monexamine.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#ifndef MONEXAMINE
+#define MONEXAMINE
+
+class monster;
+
+namespace monexamine
+{
+void milk_source( monster *source_mon );
```

Comment:
You don't check if the `source_mon` is ever `nullptr` and don't expect `nullptr` arguments here, so a reference would be better here.